core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -49,13 +55,21 @@ oppia.factory('explorationData', [
     // Put exploration variables here.
     var explorationData = {
       explorationId: explorationId,
+      // Note that the changeList is the full changeList since the last
+      // commited version.
```

Comment:
committed (typo)

And, maybe to make it clearer: `since the last committed version (as opposed to the most recent autosave)`.